src/php/README.md

```diff
@@ -60,8 +84,8 @@ $ make
 $ sudo make install
 ```
 
-In your php.ini file, add the line `extension=grpc.so` to load the extension
-at PHP startup.
+(Optional) In your php.ini file, add the line `extension=grpc.so` to load
```

Comment:
Two things:
 * all our scripts have a `-d extension=grpc.so` flag, so doing this will actually emit a warning.
 * I have a work item to add instructions for testing with Apache, which will probably touch on this again.